ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,175 +1,180 @@
+- `RegisterL1ValidatorMessage`
+- `L1ValidatorRegistrationMessage`
+- `L1ValidatorWeightMessage`
 
 The method of requesting signatures for these messages is left unspecified. A viable option for supporting this functionality is laid out in [ACP-118](../118-warp-signature-request/README.md) with the `SignatureRequest` message.
 
 All node IDs contained within the message specifications are represented as variable length arrays such that they can support new node IDs types should the P-Chain add support for them in the future.
 
 The serialization of each of these messages is as follows.
 
-#### `SubnetConversionMessage`
+#### `SubnetToL1ConversionMessage`
 
-The P-Chain can produce a `SubnetConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
+The P-Chain can produce a `SubnetToL1ConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
 
-The following serialization is defined as a `ValidatorData`:
+The following serialization is defined as the `ValidatorData`:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `nodeID` | `[]byte` | 4 + len(`nodeID`) bytes |
-| `blsPublicKey` | `[48]byte` | 48 bytes |
-| `weight` | `uint64` | 8 bytes |
-| | | 60 + len(`nodeID`) bytes |
+|          Field |       Type |                     Size |
+| -------------: | ---------: | -----------------------: |
+|       `nodeID` |   `[]byte` |  4 + len(`nodeID`) bytes |
+| `blsPublicKey` | `[48]byte` |                 48 bytes |
+|       `weight` |   `uint64` |                  8 bytes |
+|                |            | 60 + len(`nodeID`) bytes |
 
+The following serialization is defined as the `ConversionData`:
 
-The following serialization is defined as the `SubnetConversionData`:
-
-| Field | Type | Size |
-| -: | -: | -: |
-| `codecID`  | `uint16` | 2 bytes |
-| `subnetID` | `[32]byte` | 32 bytes |
-| `managerChainID` | `[32]byte` | 32 bytes |
-| `managerAddress` | `[]byte` | 4 + len(`managerAddress`) bytes |
-| `validators` | `[]ValidatorData` | 4 + sum(`validatorLengths`) bytes |
-| | | 74 + len(`managerAddress`) + len(`validatorLengths`) bytes |
+|            Field |              Type |                                                       Size |
+| ---------------: | ----------------: | ---------------------------------------------------------: |
+|        `codecID` |          `uint16` |                                                    2 bytes |
+|       `subnetID` |        `[32]byte` |                                                   32 bytes |
+| `managerChainID` |        `[32]byte` |                                                   32 bytes |
+| `managerAddress` |          `[]byte` |                            4 + len(`managerAddress`) bytes |
+|     `validators` | `[]ValidatorData` |                          4 + sum(`validatorLengths`) bytes |
+|                  |                   | 74 + len(`managerAddress`) + len(`validatorLengths`) bytes |
```

Comment:
Unrelated change - but we should probably clean it up either here or in a follow-up.
```suggestion
|                  |                   | 74 + len(`managerAddress`) + sum(`validatorLengths`) bytes |
```